### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by El-Fitz**
Seems like all that is need to ensure Carthage Support is to remove the SwiftLint Run Script.

### CODE REVIEW

**Comment by tbaranes** [comment]
🤔Why do you need to remove SwiftLint? What error did you get? SwiftLint is still useful, I'd like to keep the script if that's possible.

Also, could you update the README by adding `Carthage` badge and a `Carthage` section in `Installation`?

File: VersionTracker/VersionTracker.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -290,7 +290,6 @@
 				E2D4BF241D1E878F007A4E68 /* Frameworks */,
 				E2D4BF251D1E878F007A4E68 /* Headers */,
 				E2D4BF261D1E878F007A4E68 /* Resources */,
-				E2D4BFC31D1E8AFB007A4E68 /* Run Script - ⚠️ SwiftLint */,
```
Comment (tbaranes) [inline VersionTracker/VersionTracker.xcodeproj/project.pbxproj:293]: Why is it necessary? What error did you get? 
The script is still useful, I rather prefer to keep it

**Review by tbaranes — CHANGES_REQUESTED** [review]
Could you also: 
- Update the README by adding `Carthage` badge and a `Carthage` section in `Installation`?
- Add an entry in the CHANGELOG

File: VersionTracker/VersionTracker.xcodeproj/project.pbxproj (hunk 1 above)
Comment (El-Fitz) [reply VersionTracker/VersionTracker.xcodeproj/project.pbxproj:293]: Well, I got 7 lines of

> warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint

Which, now that I'm writing it down, was just a warning. Nothing more.
So I'm stupid. Like... really stupid. 😓

Sorry, used to get 0's when we the compiler issued a warning.

**Comment by El-Fitz** [comment]
Will update the README and the changelog though :-)

**Pull request closed without merging**